### PR TITLE
Announcement & CSS

### PR DESCRIPTION
**Pull request by drowsy810301**
- add announcement in index page
- modify some index's css to match every theme

### CODE REVIEW

File: index/models.py [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+    SOFTWARE.
+    '''
 from django.db import models
-
+from datetime import datetime
 # Create your models here.
+
+class Announcement(models.Model):
+
+    content = models.TextField(null=True)
```
Comment (nickelcheng) [inline index/models.py]: I don't prefer a null announcement.

**Comment by nickelcheng** [comment]
+1 besides the comment. @bruce3557 please check

**Comment by drowsy810301** [comment]
comment fixed

**Comment by drowsy810301** [comment]
comment fixed

**Comment by drowsy810301** [comment]
comment fixed

**Comment by nickelcheng** [comment]
Where can I add an announcement?

**Comment by nickelcheng** [comment]
Please add buttons for add / edit / delete announcements.
If there is no active announcement, please hide the title.

**Comment by drowsy810301** [comment]
lastest dev merged
conflict fixed

**Comment by drowsy810301** [comment]
can add,edit and delete announcement

**Comment by drowsy810301** [comment]
I think we still need the title when there is no announcement beacause admin would not know where to add an announcement.

**Comment by drowsy810301** [comment]
datetimepicker need pip install django-datetime-widget,please review the code again,thanks.


screenshot from 2015-04-03 01 33 42

**Comment by nickelcheng** [comment]
+1, That's great! @bruce3557 please check

File: index/models.py [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+'''
+from django.db import models
+from datetime import datetime
 # Create your models here.
+
+class Announcement(models.Model):
+
+    content = models.TextField(default=None)
+    start_time = models.DateTimeField(default=datetime.now)
+    end_time = models.DateTimeField(default=datetime.now)
```
Comment (bruce3557) [inline index/models.py]: should we consider whose own the announcement?

Comment (drowsy810301) [inline index/models.py]: @nickelcheng

Comment (nickelcheng) [inline index/models.py]: It must be admin user, I think we don't have to do that.

File: index/templates/index/base.html [hunk 3]
```diff
@@ -125,111 +126,119 @@
...
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                <p class="p2">Team<b class="caret"></b></p>
+              </a>
+              <ul class="dropdown-menu">
+                <li><a href="{% url 'team:list' %}">Team List</a></li>
+                <li><a href="{% url 'team:profile' %}">Team Profile</a></li>
+              </ul>
+            </li>
+            -->
```
Comment (bruce3557) [inline index/templates/index/base.html]: Please remove comments

File: index/templates/index/base.html [hunk 4]
```diff
@@ -125,111 +126,119 @@
       </div>
       <div id="mwt_slider_content">feedback</div>
     </div>
+    -->
```
Comment (bruce3557) [inline index/templates/index/base.html]: Please remove comments

File: users/templates/users/profile.html [hunk 5]
```diff
@@ -27,7 +27,7 @@
 {% load profile_filters %}
 
 {% block body_block %}
-
+<br>
```
Comment (bruce3557) [inline users/templates/users/profile.html]: not sure why add a new line here

**Comment by bruce3557** [comment]
+1 besides the comments

File: index/templates/index/base.html (hunk 4 above)
Comment (nickelcheng) [inline index/templates/index/base.html]: The feedback function will be complete latter. I prefer not to remove.

File: users/templates/users/profile.html (hunk 5 above)
Comment (drowsy810301) [inline users/templates/users/profile.html]: because it is to close to the navbar

Comment (bruce3557) [inline users/templates/users/profile.html]: fine

**Comment by drowsy810301** [comment]
latest code merged

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**